easybuild/easyconfigs/c/cdat/cdat-8.2.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,132 @@
+        'sources': ['%(name)s-%(version)s.tar.gz'],
+        'checksums': ['5944a9b95e97de1980c65f03b79b356f30a43de48682b8bdd90aa5089f0ec1f4'],
+    }),
+    ('eofs', '1.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/%(nameletter)s/%(name)s'],
+        'sources': ['%(name)s-%(version)s.tar.gz'],
+        'checksums': ['5ae9afc159b8cfb2be476d257fc469b2cdd473c76f5411c508010007a5ae6bd2'],
+    }),
+    ('cdtime', '3.1.4', {
+        'checksums': ['c22ec725e207994cec3d4ba5b5b35ede5ea698d552dc75e87cd86fa7dbc65b35'],
+    }),
+    ('cdutil', version, {
+        'checksums': ['9e02348ae0e9148c6b0b9391b208adf9be7d97e0da30b88518c02dfc1e07d2bf'],
+    }),
+    ('dv3d', version, {
+        'modulename': 'DV3D',
+        'patches': ['dv3d-pkg_resource-paths.patch'],
+        'checksums': [
+            'd542641e259327777642616def558ea97ffdac5788f7258348812534a89f01a3',  # dv3d-8.2.1.tar.gz
+            '3d2dde578998dcae9a45195b495cc9b335d706d9b1cea9e766067af6a00cdd98',  # dv3d-pkg_resource-paths.patch
+        ],
+    }),
+    ('vcs', version, {
+        'checksums': ['9f647c940396cb40003b96484bfa48d2d0fce38cf49217e06054c4c9cc66b5ce'],
+    }),
+    ('vcsaddons', version, {
+        'checksums': ['b8d431243ee541da46f4e5e19c21eb911214c597958ab237d9484854f173b6e3'],
+    }),
+    ('xmgrace', '2.10', {
+        'checksums': ['6d5b80d58807b39722895947a61a5e900bfac0d1f70c86381846b736c0e2ef20'],
+    }),
+    ('genutil', version, {
+        'checksums': ['944ccc4de05072d2773cfff6b5f279e2058457a672169153ebec0e19f5a3601c'],
+    }),
+    ('thermo', '8.1', {
+        'checksums': ['9bc854a321d7ad1b46076185ccb5b7e32cbc595794f691526e24770c52eae01b'],
+    }),
+    ('wk', version, {
+        'modulename': 'WK',
+        'checksums': ['c060f0e31340b139c86bac9a49b9e06136e63c48c75d5745080312cda3e79ff5'],
+    }),
+    ('distarray', '2.12.2', {
+        'sources': [{'filename': '%(name)s-%(version)s.tar.gz', 'download_filename': '%(version)s.tar.gz'}],
+        'checksums': ['ca6ee7610b8e8309cec47121b2454d8d318bfe37b6db3b950c65b1316035ba6c'],
+    }),
+    ('cdms2', '3.1.5', {
+        'patches': ['cdms2-3.1.5-intel-2020a.patch'],
+        'source_urls': ['https://github.com/CDAT/cdms/archive/'],
+        'checksums': [
+            '2b4f2ad348581c9831d3dcb785502515da41f0d21be728c8211618871a0caed9',  # cdms2-3.1.5.tar.gz
+            '378aa4da96a2f15d1d156a9c0ca80ab93d533379a6d30b5c3188b321730008d0',  # cdms2-3.1.5-intel-2020a.patch
+        ],
+    }),
+]
+
+postinstallcmds = [
+    # Generated in pycf's setup.py
+    'sed -i "s/ extern//g" %(installdir)s/lib/python%(pyshortver)s/site-packages/pycf/libCFConfig.py',
+]
+
+use_pip = True
+download_dep_fail = True
+
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ['python -c "import genutil;from eofs.cdms import Eof;import vcs;import MV2"']
```

Comment:
`import` for `genutil` and `vcs` is already done automatically, since they're listed as extensions.

`from eofs.cdms import Eof` is more specific than `import eofs`, so it makes sense to retain that.

Which part provides `MV2`? Worth clarifying with a comment?

Reply:
Oops, leftover from debugging, I took a snippet of a CDAT script.  MV2 is provided by CDMS2, I believe, but the implicit PythonPackage sanity checks take care of this quite well.